apps/documents/views.py

```diff
@@ -38,9 +38,8 @@ class DocumentViewSet(mixins.CreateModelMixin,
     def get_queryset(self):
         queryset = self.queryset
         if settings.PROFILES_ENABLED:
-            shipment_id = self.request.parser_context['kwargs'].get('shipment_pk', None)
-            if shipment_id:
-                return queryset.filter(shipment__id=shipment_id)
+            shipment_id = self.request.parser_context['kwargs']['shipment_pk']
```

Comment:
I am fairly certain you can use `self.kwargs` or `self.request.kwargs` to get the shipment_pk.